PlanGrid.Api/Project.cs

```diff
@@ -10,5 +10,41 @@ public class Project : Record
     {
         [JsonProperty("name")]
         public string Name { get; set; }
+
+        [JsonProperty("custom_id")]
+        public string CustomId { get; set; }
+
+        [JsonProperty("type")]
+        public string Type { get; set; }
+
+        [JsonProperty("status")]
+        public string Status { get; set; }
+
+        [JsonProperty("owner")]
+        public string Owner { get; set; }
+
+        [JsonProperty("start_date")]
+        public DateTime? StartDate { get; set; }
```

Comment:
Ah, good point.  So they never include a time component?  In that case, yeah, @JuliusAlexanderIV is right and they should use our custom `Date` class instead.  Also, I second my thanks! :)